### PR TITLE
Fix rare crash of appleseed on exit

### PR DESCRIPTION
**Pull request by dictoon**

### CODE REVIEW

File: src/appleseed/renderer/kernel/intersection/intersector.cpp [hunk 1]
```diff
@@ -1,389 +1,389 @@
...
-
+
+//
+// This source file is part of appleseed.
+// Visit https://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2010-2013 Francois Beaune, Jupiter Jazz Limited
+// Copyright (c) 2014-2018 Francois Beaune, The appleseedhq Organization
```
Comment (oktomus) [inline src/appleseed/renderer/kernel/intersection/intersector.cpp]: 2019

**Comment by dictoon** [comment]
> It's hard to see what changed in intersector.cpp. It seems you changed the order of some declarations

Line endings were busted, I fixed them. Nothing else has changed.

File: src/appleseed/renderer/kernel/intersection/intersector.cpp (hunk 1 above)
Comment (dictoon) [reply src/appleseed/renderer/kernel/intersection/intersector.cpp]: At some point I'll do a pass on all files to update copyright notices. Usually I do that at the beginning of each year but this time I've been lazy. Might as well wait until January 2020 at this point :)

**Comment by dictoon** [comment]
Thanks for the review @oktomus!

**Pull request merged**